packages/tachyons-background-size/css/tachyons-background-size.css

```diff
@@ -1,22 +1,27 @@
 /*!!!
 
-  # BACKGROUND SIZE
+# BACKGROUND SIZE
 
-  Often used in combination with background image set as an inline style
-  on an html element.
+Often used in combination with background image set as an inline style
+on an html element.
 
-  ### Docs
+### Docs
 
-  http://tachyons.io/docs/themes/background-size/
+http://tachyons.io/docs/themes/background-size/
 
-  ### Media Query Extensions
+### Media Query Extensions
 
```

Comment:
This seems to be missing the `base` section?

Reply:
Fixt!